### PR TITLE
Added timeout options

### PR DESCRIPTION
**Pull request by yyjim**
## Description

Our Figma file has too many assets. The default timeout is not enough for us to get all assets.

## Changes

- Add timeout option to change default request timeout.

### CODE REVIEW

**Comment by yyjim** [comment]
Hello @subdan  
Please help to review this PR when you have time. Thanks

File: CONFIG.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ figma:
   lightFileId: shPilWnVdJfo10YF12345
   # [optional] Identifier of the file containing dark color palette and dark images.
   darkFileId: KfF6DnJTWHGZzC912345
+  # [optional] Figma API request timeout, default is 30s
```
Comment (subdan) [inline CONFIG.md:18]: Add more info why a user may want to change the value. For example:
`# [optional] Figma API request timeout. The default value of this property is 30 (seconds). If you have a lot of resources to export set this value to 60 or more to give Figma API more time to prepare resources for exporting.`

Comment (yyjim) [reply CONFIG.md]: 👍  I updated it.

**Review by subdan — APPROVED** [review]

**Comment by subdan** [comment]
Thanks for the PR, @yyjim! The new version released.
https://github.com/RedMadRobot/figma-export/releases/tag/0.18.8

**Comment by yyjim** [comment]
👍  Thank you.

**Pull request merged**